docs/conf.py

```diff
@@ -160,6 +160,8 @@
     # TODO: remove once we support different rtd config
     # files per project.
     "conf_py_path": f"/docs/{docset}/",
+    # Use to generate the Plausible "data-domain" attribute from the template
+    "plausible_domain": f"{os.environ.get('READTHEDOCS_PROJECT')}.readthedocs.io",
```

Comment:
Running a normal `PROJECT=user make html` locally, I get:

```
<script defer data-domain="None.readthedocs.io" src="[https://plausible.io/js/script.js](view-source:https://plausible.io/js/script.js)"></script>
```

But it works for me with `docset`

```suggestion
    "plausible_domain": f"{docset}.readthedocs.io",
```

Reply:
I've done that first. The problem with that is that it doesn't match the subdomain. `docset` values are `user` and `dev` and we need `docs` and `dev`. The project slug is the right value.

Locally, this value doesn't make too much sense. We could remove plausible locally if we want to, tho.

Reply:
> Locally, this value doesn't make too much sense. We could remove plausible locally if we want to, tho.

I did this by checking for `if READTHEDOCS` in the template.

Reply:
Oh right, I forgot that we have `user` matching `docs` here :)

The new solution is better :+1: